src/ts/components/account/Term.tsx

```diff
@@ -13,8 +13,9 @@ class Term extends React.Component<ITermProp> {
 
   handleClick = () => {
     const { history, settings } = this.props
-    this.props.saveSettings({ ...settings, welcome: false })
-    history.push(CREATE_PASSWORD_ROUTE)
+    this.props.saveSettings({ ...settings, welcome: false }).then(() =>
+      history.push(DEFAULT_ROUTE)
```

Comment:
Does that mean now the `DEFAULT_ROUTE` will now check if it goes to login or create new password?

Reply:
yeah i probably should rename it to home_route